common/components/BalanceSidebar/AccountInfo.tsx

```diff
@@ -102,13 +102,23 @@ class AccountInfo extends React.Component<Props, State> {
         <h5 className="AccountInfo-section-header">{translate('SIDEBAR_ACCOUNTADDR')}</h5>
         <div className="AccountInfo-section AccountInfo-address-section">
           <div className="AccountInfo-address-icon">
-            <Identicon address={address} size="100%" />
+            <Identicon
+              address={toChecksumAddressByChainId(address, network.chainId)}
```

Comment:
Rather than checksum every time for Identicon props, perhaps Identicon should do the checksumming.